packages/medusa/src/models/batch-job.ts

```diff
@@ -18,10 +19,17 @@ export class BatchJob extends SoftDeletableEntity {
   created_by_user: User
 
   @DbAwareColumn({ type: "jsonb", nullable: true })
-  context: { retry_count?: number; max_retry?: number } & Record<string, unknown>
+  context: {
+    retry_count?: number
+    max_retry?: number
+  } & RequestQueryFields & Record<string, unknown>
```

Comment:
comment: I realize, that if we choose to snake_case all properties as suggested, we will not be able to use the `RequestQueryFields` type unless we also modify that which is prob. not possible if it's used elsewhere. We could create a new type - let me know what you think?

Reply:
Yes, I will certainly create a new type, specially that it could defer from that one since they do not serve the same purpose :)